hackathon/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.urls import path
-from .views import (
-    HackathonListView, create_hackathon, update_hackathon, delete_hackathon, judging
-)
+from .views import *
 
 urlpatterns = [
     path('', HackathonListView.as_view(), name="hackathon-list"),
     path("<int:hack_id>/team/<int:team_id>/judging/", judging, name="judging"),
     path("create_hackathon", create_hackathon, name='create_hackathon'),
-    path("<int:hackathon_id>/update_hackathon", update_hackathon, name="update_hackathon"),
-    path("<int:hackathon_id>/delete_hackathon", delete_hackathon, name="delete_hackathon"),
+    path("<int:hackathon_id>/view_hackathon", view_hackathon,
```

Comment:
Can we just make the detail view have a path of `<int:hackathon_id>` so that the route will be `https://host/hackathon/<int:hackathon_id>`? That would look a bit better I think.

Reply:
Done